test/README.md

```diff
@@ -158,26 +158,26 @@ rhsm = true
 ### Image Blueprints
 
 The image blueprints are independent of the test scenarios so that
-images can be reused. Be careful making changes to specific images in
-case the change affects the way the image is used in different
-scenarios. Be careful adding unnecessary new images, since each image
-takes time to build and may slow down the overall test job.
-
-Images are organized into "groups". Each group is built in order and
-all of the images in a group are built in parallel.
-
-Add blueprints as TOML files in the appropriate group directory in the
-`./test/image-blueprints` directory, then add a short description of the
-image here for reference.
-
-Blueprint | Group | Image Name | Purpose
---------- | ----- | ---------- | -------
-rhel92.toml | group1 | rhel-9.2 | A simple RHEL image without MicroShift.
-rhel92-microshift-previous-minor.toml | group2 | rhel-9.2-microshift-4.13 | A RHEL 9.2 image with the latest MicroShift from the previous y-stream installed and enabled.
-rhel92-source.toml | group2 | rhel-9.2-microshift-source | A RHEL 9.2 image with the RPMs built from source.
-rhel92-source-fake-next-minor.toml | group2 | rhel-9.2-microshift-4.15 | A RHEL 9.2 image with the RPMs built from source from the current PR but with the _version_ set to the next y-stream.
-rhel92-source-fake-yplus2-minor.toml | group2 | rhel-9.2-microshift-4.16 | A RHEL 9.2 image with the RPMs built from source from the current PR but with the _version_ set to the current+2 y-stream.
-rhel92-crel.toml | group2 | rhel-9.2-microshift-crel | A RHEL 9.2 image with MicroShift RPMs already built and released by ART like ECs, RCs, and Z-stream releases.
+images can be reused.
+
+Images are organized into "layers" that contain "groups".
+
+Each layer has its designation:
+- `base` layer is a prerequisite for all the subsequent ones
+- `presubmit` layer contains images used in presubmit CI jobs
+- `periodic` layer contains images used in periodic CI jobs
+
+Each group in a given layer is built in order and all of the images in
```

Comment:
Does this mean it builds each group of each layer in order as a separate step

base group 1
base group 2
base group 3
presubmit group 1
presubmit group 2

or does it build all of the same group across all layers at the same time

group 1 from base and presubmit
group 2 from base and presubmit

?

Reply:
I have not implemented parallelization on the level of job type yet, so it's the former case. We can apply this optimization in the future if we see it's necessary. 
I changed the wording in the doc to make it more explicit. Is this better now?